jetstream/consumer.ts

```diff
@@ -314,7 +314,7 @@ export class PullConsumerMessagesImpl extends QueuedIteratorImpl<JsMsg>
     super();
     this.consumer = c;
 
-    const copts = opts as ConsumeOptions;
+    const copts = { ...opts } as ConsumeOptions;
```

Comment:
would it be better to do `structuredClone()` in case some of those structures ever become objects, etc.

Reply:
some of those will take a callback - so cannot use structuredClone()...